pkg/controller/plan/kubevirt.go

```diff
@@ -1224,7 +1244,7 @@ func (r *KubeVirt) guestConversionPod(vm *plan.VMStatus, vmVolumes []cnv.Volume,
 		},
 		Spec: core.PodSpec{
 			SecurityContext: &core.PodSecurityContext{
-				FSGroup: &fsGroup,
+				FSGroup: &qemuUser,
 			},
```

Comment:
You added `SeccompProfile` to the PVC binding Pod but forgot to add it to the conversion Pod.

Reply:
i really don't like the mixup of `SeccompProfile` when setting user ID. let's not add it.